proxmoxtf/resource/container.go

```diff
@@ -331,23 +332,31 @@ func Container() *schema.Resource {
 							Type:        schema.TypeBool,
 							Description: "Whether the container runs as nested",
 							Optional:    true,
-							ForceNew:    true,
 							Default:     dvResourceVirtualEnvironmentContainerFeaturesNesting,
 						},
 						mkResourceVirtualEnvironmentContainerFeaturesKeyControl: {
 							Type:        schema.TypeBool,
 							Description: "Whether the container supports `keyctl()` system call",
 							Optional:    true,
-							ForceNew:    true,
 							Default:     dvResourceVirtualEnvironmentContainerFeaturesKeyControl,
 						},
 						mkResourceVirtualEnvironmentContainerFeaturesFUSE: {
 							Type:        schema.TypeBool,
 							Description: "Whether the container supports FUSE mounts",
 							Optional:    true,
-							ForceNew:    true,
 							Default:     dvResourceVirtualEnvironmentContainerFeaturesFUSE,
 						},
+						mkResourceVirtualEnvironmentContainerFeaturesMountTypes: {
+							Type:        schema.TypeList,
+							Description: "List of allowed mount types",
+							Optional:    true,
+							Elem: &schema.Schema{
+								Type:             schema.TypeString,
+								ValidateDiagFunc: validator.MountType(),
+							},
+							DiffSuppressFunc:      structure.SuppressIfListsAreEqualIgnoringOrder,
```

Comment:
When I apply 
```hcl
features {
  mount = ["nfs", "cifs"]
}
```
and then comment `mount` and reapply, TF shows "no changes". Not sure why. Let me see if there is a bug in `SuppressIfListsAreEqualIgnoringOrder` 🤔

Reply:
Yeah, we should remove `DiffSuppressFunc` and `DiffSuppressOnRefresh` attributes altogether. 

We use them for `tags` because PVE always sorts tags alphabetically, but that's not the case for mount types. And I have a sneaky suspicion that `DiffSuppressFunc` is not the right way to address the sorting issue, `StateFunc` seems to be a better alternative for that.